docs/get-started/example-solutions/_index.rst

```diff
@@ -55,29 +55,6 @@ For an introduction to using the training APIs, please visit :ref:`Training APIs
       -  Enron Email Corpus
       -  :download:`gpt_neox.tgz </examples/gpt_neox.tgz>`
 
-********************
- DeepSpeed Autotune
-********************
-
-.. list-table::
-   :header-rows: 1
-
-   -  -  Framework
-      -  Dataset
-      -  Filename
-
-   -  -  DeepSpeed (PyTorch)
-      -  ImageNet (Generated)
-      -  :download:`torchvision.tgz </examples/torchvision.tgz>`
-
-   -  -  Hugging Face (DeepSpeed/PyTorch)
-      -  Beans (Hugging Face)
-      -  :download:`hf_image_classification.tgz </examples/hf_image_classification.tgz>`
-
-   -  -  Hugging Face (DeepSpeed/PyTorch)
-      -  WikiText (Hugging Face)
-      -  :download:`hf_language_modeling.tgz </examples/hf_language_modeling.tgz>`
```

Comment:
I don't think these should be deleted, but they should be recategorized as plain DeepSpeed examples.  Or maybe the last two could be classified as HF examples.

However, within these examples, there are dsat details that need stripping out.  (I forwarded those to you in slack)

Reply:
done